_posts/en/newsletters/2024-01-31-newsletter.md

```diff
@@ -0,0 +1,253 @@
+  executing an `OP_CHECKSEQUENCEVERIFY` opcode appears to have a
+  negative version number.  This test, if it had been run by alternative
+  consensus implementations, would have caught the consensus failure bug
+  mentioned in [last week's newsletter][news286 bip68ver].
+
+- [Eclair #2811][], [#2813][eclair #2813], and [#2814][eclair #2814]
+  adds the ability for a [trampoline payment][topic trampoline payments]
+  to use a [blinded path][topic rv routing] for the ultimate receiver.
+  Trampoline routing itself continues to use regular onion-encrypted
+  node IDs, i.e. each trampoline node learns the ID for the next
+  trampoline node.  However, if a blinded path is used, the final
+  trampoline node will now only learn the node ID for the first
+  forwarding node in the blinded path; it will not learn the ID for
+  the ultimate receiver.
+
+    Previously, strong trampoline privacy depended on using multiple
+    trampoline forwarders so that none of the forwarders could be sure
+    they were the final forwarder.  A downside of this approach is that
+    it used longer paths that increased the probability of forwarding
+    failure and required paying more forwarding fees for success.  Now
+    forwarding payments through even a single trampoline node can
+    prevent that node from learning the ultimate receiver.
+
+- [LND #8167][] allows an LND channel to mutually close a channel that
+  still has one or more pending payments ([HTLCs][topic htlc]).  The
+  [BOLT2][] specification indicates the proper procedure for this is for
+  one side to send a `shutdown` message, after which no new HTLCs will
+  be accepted.  After all pending HTLCs are resolved offchain, the two
+  parties negotiate and sign a mutual close transaction.  Previously,
+  when LND received a `shutdown` message, it would force close the
+  channel, requiring extra onchain transactions and fees to settle.
+
+- [LND #7733][] updates [watchtower][topic watchtowers] support to enable
+  backing up and enforcing correct shutdown of the [simple taproot
+  channels][topic simple taproot channels] that are now supported
+  experimentally by LND.
+
+- [LND #8275][] begins requiring peers support certain
+  universally-deployed features as specified in [BOLTs #1092][] (see
+  [Newsletter #259][news259 lncleanup]).
+
+- [Rust Bitcoin #2366][] depreciates the `.txid` property on
+  `Transaction` objects and begins providing a replacement property
+  named `.compute_txid`.  Each time the `.txid` property is called, the
+  txid for the transaction is calculated, which consumes enough
+  CPU for it to be a concern to anyone running the function on large
+  transactions or many smaller transactions.  It is hoped that the new
+  name for the property will help downstream programmers realize its
+  potential costs.  The `.wtxid` and `.ntxid` properties (respectively
+  based on [BIP141][] and [BIP140][]) are similarly renamed to
+  `.compute_wtxid` and `.compute_ntxid`.
```

Comment:
To be more precise, all these are _methods_ on `Transaction`, not properties. And as such they should be accompanied by parentheses, e. g. `.compute_txid()`.